.github/workflows/common-workflows.yaml

```diff
@@ -12,6 +12,6 @@ jobs:
     uses: dell/common-github-actions/.github/workflows/go-static-analysis.yaml@main
     name: Golang Validation
 
-  common:
-    name: Quality Checks
-    uses: dell/common-github-actions/.github/workflows/go-common.yml@main
+#  common:
+#    name: Quality Checks
+#    uses: dell/common-github-actions/.github/workflows/go-common.yml@main
```

Comment:
Shouldn't these commented out lines be removed?

Reply:
When I initally committed with no code changes, the workflow UT failed. The new unit-test-workflow.yaml takes the place of these lines. If there is an alternative that uses the common workflow and excludes the directories on line 31 in the unit-test-workflow.yaml, then I can change this.